PWGCF/MultiparticleCorrelations/Tasks/multiparticle-correlations-ar.cxx

```diff
@@ -13,19 +13,56 @@
 /// \brief multiparticle-correlations-ar - Task belonging to Anton Riedel for computing multiparticle correlations
 /// \author Anton Riedel, TU München, anton.riedel@tum.de
 
-#include <fairlogger/Logger.h>
+#include "fairlogger/Logger.h"
+#include <cstdint>
+#include <string_view>
+#include <string>
+#include <vector>
+#include <array>
+#include <numeric>
+#include "TComplex.h"
 #include "Framework/runDataProcessing.h"
 #include "Framework/AnalysisTask.h"
+#include "Framework/Expressions.h"
 #include "Common/DataModel/Centrality.h"
 #include "Common/DataModel/Multiplicity.h"
 #include "Common/DataModel/TrackSelectionTables.h"
 
 using namespace o2;
 using namespace o2::framework;
+using namespace o2::framework::expressions;
 
-namespace MultiParticleCorrelationsARTaskGlobalConfig
+// Define useful constants and enums in a separate name space
+namespace MultiParticleCorrelationsARTaskGlobalVariables
 {
-// setup for event variables
+// for before and after applying cuts
+enum BA {
+  kBEFORE,
+  kAFTER,
+  kLAST_BA
+};
+static constexpr std::string_view BeforeAfter[kLAST_BA] = {"before/", "after/"};
+
+// for reconstructed and simulated data
+enum RS {
```

Comment:
What about "RecoSim"?

Reply:
or "RecoSimEnum", when RecoSim is already taken?

Reply:
Fixed.